Bookmark-Manager/src/main/java/com/bookmarkmanager/bookmark/BookMarkController.java

```diff
@@ -0,0 +1,46 @@
+package com.bookmarkmanager.bookmark;
+
+
+import com.bookmarkmanager.pojo.Bookmark;
+import lombok.RequiredArgsConstructor;
+
+import java.net.URI;
+import java.util.UUID;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/bookmarks")
+@RequiredArgsConstructor
+public class BookMarkController {
+  private final BookmarkService bookmarkService;
+
+
+  // Create a new bookmark
```

Comment:
Comments not needed, We add comments when we have complex logic

Reply:
sure.